python-package/lightgbm/sklearn.py

```diff
@@ -993,6 +992,6 @@ def fit(self, X, y,
     _base_doc = fit.__doc__
     _before_early_stop, _early_stop, _after_early_stop = _base_doc.partition('early_stopping_rounds :')
     fit.__doc__ = (_before_early_stop
-                   + 'eval_at : iterable of int, optional (default=(1, 2, 3, 4, 5))\n'
-                   + ' ' * 12 + 'The evaluation positions of the specified metric.\n'
-                   + ' ' * 8 + _early_stop + _after_early_stop)
+                   "eval_at : iterable of int, optional (default=(1, 2, 3, 4, 5))\n"
```

Comment:
ah! I'm so sorry @akshitadixit , I misled you in my previous comment. To concatenate like this, I guess you do need to do `f"{_before_early_stop}"`. It isn't possible to concatenate a variable and a string literal like this.

```python
a = "hello"
x = (
    a
    " world"
)
```

Could you please change `_before_early_stop` to `f"{_before_early_stop}"`?

Sorry again, totally my fault for asking you to change that.

Reply:
Not a problem, I did think of checking it up but then laid back on it.